app/controllers/figures_controller.rb

```diff
@@ -1,13 +1,21 @@
+# frozen_string_literal: true
```

Comment:
@jennceng We had many hound violation that said something like "Frozen string literal" and this was the fix we found